lib/gems/pending/util/extensions/miq-benchmark.rb

```diff
@@ -29,6 +29,10 @@ def self.realtime_block(key, &block)
     begin
       ret = realtime_store(hash, key, &block)
       return ret, hash
+    rescue
+      # Don't let timings be lost on exception when there is nobody else to pick them up.
+      $log.info("Exception in realtime_block #{key.inspect} - Timings: #{hash.inspect}") if outermost
```

Comment:
The Benchmark extension is meant to be non-miq-specific.  In fact, it's on the list to be moved to more_core_extensions.  Imagine it's in more_core, how would this change work?